src/ModelerApp.vue

```diff
@@ -18,9 +18,9 @@
           </b-btn>
         </div>
       </b-card-header>
-
-      <modeler ref="modeler" @validate="validationErrors = $event" @warnings="warnings = $event" />
-
+      <b-card-body class="overflow-hidden position-relative p-0 vh-100">
```

Comment:
When this is in the modeler component, the panel scrollbars seem go away in modeler and partially work in processmaker.